func_adl_xAOD/atlas/xaod/translator.py

```diff
@@ -0,0 +1,48 @@
+import func_adl_xAOD.atlas.xaod.event_collections as event_collections  # NOQA
```

Comment:
This filename isn't good. How about query_visitor or simliar?

Reply:
And also track the naming convention you use on the cms side to (e.g. cms as part of it).